### DISCUSSION TITLE
Solana Tech Discord

### DISCUSSION THREAD

**Discussion by ripatel-fd** [firedancer-io/firedancer#889] [Announcements]
If you'd like to reach out to the Firedancer team directly, you may find the Solana Tech Discord at https://solana.com/discord
The Solana Tech moderators have provided a `#firedancer-operators` channel intended for technical discussion and support requests.

Please note that this Discord is not affiliated with the Firedancer team. It is moderated by Solana community contributors.

**Comment by hbf731eF** [comment]
there is no #firedancer-public channel in discord

**Reply by ripatel-fd** [reply]
Looks like it was recently renamed to `#firedancer-operators`.

**Comment by hoaffam** [comment]
Is there anyone here who uses macOS and can set it up?
